esmvalcore/preprocessor/_multimodel.py

```diff
@@ -107,15 +107,22 @@ def _put_in_cube(template_cube, cube_data, statistic, t_axis):
             t_axis,
             standard_name='time',
             units=template_cube.coord('time').units)
-    lats = template_cube.coord('latitude')
-    lons = template_cube.coord('longitude')
+
```

Comment:
These changes seem unrelated to deriving the variable? Would it make sense to put this in a separate pull request?